src/Evolution/DgSubcell/Tags/TciGridHistory.hpp

```diff
@@ -0,0 +1,23 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <deque>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+
+namespace evolution::dg::subcell::Tags {
+/// A record of which grid the TCI requested we use.
+///
+/// This is necessary because when using linear multistep methods for the time
+/// integration we need to wait to return to DG until the entire history is
```

Comment:
I would say "wait until ... smooth in order to return to DG" (I find that easier to parse)

Reply:
I like @kidder's suggestion, but I'll iterate and suggest "in order to return" -> "before returning".

In other words, "... wait until the entire history is smooth before returning to DG"